### DISCUSSION TITLE
updating is giving an error

### DISCUSSION THREAD

**Discussion by esuess** [dat-linux/community#29] [Issues]
It looks like there has been a change with how gitlab security works or has the link to the control panel changed?  

sudo apt update gives a 403 Forbidden error for

500 https://datlinux.gitlab.io/control-panel-1.0.1-lxqt/apt-repo stable/main amd64 Packages
     release v=1.0.143-lxqt,o=DAT Linux Repository,a=stable,n=stable,l=DAT Linux,c=main,b=amd64
     origin datlinux.gitlab.io

Err:6 https://datlinux.gitlab.io/control-panel-1.0.1-lxqt/apt-repo stable InRelease
  403  Forbidden [IP: 172.65.251.78 443]
Reading package lists... Done
E: Failed to fetch https://datlinux.gitlab.io/control-panel-1.0.1-lxqt/apt-repo/dists/stable/InRelease  403  Forbidden [IP: 172.65.251.78 443]
E: The repository 'https://datlinux.gitlab.io/control-panel-1.0.1-lxqt/apt-repo stable InRelease' is no longer signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

**Comment by dat-linux** [comment]
Apologies for the bug, please try again.
